packages/react/src/components/loaded/index.tsx

```diff
@@ -0,0 +1 @@
+export { SlashIDLoaded } from './loaded'
```

Comment:
I'm happy to change it and drop the barrel file, again not taking a strong stance here this time - just trying to follow what I perceived as convention and get my PR merged 😅 

`file-upload` does this pattern, where as `dropdown` and `divider` don't so I wasn't sure.

Reply:
Yep some of the components were not written by us - we got them with the KYC SDK, did not refactor anything at the time since the code is currently unused.

Reply:
done!

Reply:
Thanks for extra context 🙏